src/Functions/degrees.cpp

```diff
@@ -0,0 +1,27 @@
+#include <Functions/FunctionFactory.h>
+#include <Functions/FunctionMathUnary.h>
+
+template <typename T>
+T degrees(T r)
```

Comment:
Most likely the function should always return Float64, not T.
Otherwise it will be wrong for integers.
Please add a test.

Reply:
I'll update.

Reply:
thanks! updated to return type to Float64 and added more tests to ensure that results are correct for integers.

Reply:
Thanks. Changed to return Float64 instead of T. Added more tests.